omrmakefiles/rules.linux.mk

```diff
@@ -155,7 +150,16 @@ endif
 ## Debugging Information
 # Indicate that GNU debug symbols are being used
 ifeq (gcc,$(OMR_TOOLCHAIN))
-  ifneq (,$(filter arm ppc s390 x86,$(OMR_HOST_ARCH)))
+  ifeq (arm,$(OMR_HOST_ARCH))
```

Comment:
Did we lose s390 from this list?

Reply:
It would seem so. I'll be sure to put it back.